mod-source-record-storage-server/src/main/java/org/folio/consumers/ParsedRecordChunksKafkaHandler.java

```diff
@@ -97,22 +99,22 @@ public ParsedRecordChunksKafkaHandler(@Autowired RecordService recordService,
     String topicName = KafkaTopicNameHelper.formatTopicName(kafkaConfig.getEnvId(), KafkaTopicNameHelper.getDefaultNameSpace(),
       tenantId, DI_PARSED_RECORDS_CHUNK_SAVED.value());
 
-    KafkaProducerRecord<String, String> record =
+    KafkaProducerRecord<String, String> targetRecord =
       KafkaProducerRecord.create(topicName, key, Json.encode(event));
 
-    record.addHeaders(kafkaHeaders);
+    targetRecord.addHeaders(kafkaHeaders);
 
     Promise<String> writePromise = Promise.promise();
 
     String producerName = DI_PARSED_RECORDS_CHUNK_SAVED + "_Producer";
     KafkaProducer<String, String> producer =
       KafkaProducer.createShared(Vertx.currentContext().owner(), producerName, kafkaConfig.getProducerProps());
 
-    producer.write(record, war -> {
+    producer.write(targetRecord, war -> {
       producer.end(ear -> producer.close());
       if (war.succeeded()) {
-        LOGGER.debug("RecordCollection processing has been completed with response sent... correlationId {}, chunkNumber {}-{}", correlationId, chunkNumber, record.key());
-        writePromise.complete(record.key());
+        LOGGER.debug("RecordCollection processing has been completed with response sent... chunkId {}, chunkNumber {}-{}", chunkId, chunkNumber, targetRecord.key());
+        writePromise.complete(targetRecord.key());
       } else {
         Throwable cause = war.cause();
         LOGGER.error("{} write error {}", producerName, cause);
```

Comment:
Is it possible to add an additional data like chunkId and other here?

Reply:
Done